src/CEOFESABundle/Controller/MainController.php

```diff
@@ -47,14 +47,18 @@ public function dashboardAction()
      *       path="/dashboard/structure",
      *       name="structure_dashboard"
      * )
+     *
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
      */
     public function dashboardStructureAction(Request $request)
     {
         $id             = $this->get('session')->get('structure');
         $em             = $this->getDoctrine()->getManager();
         $structure      = $em->getRepository('CEOFESABundle:Structure')->find($id);
 
-        $form = $this->createForm(new DashboardType());
+        $form = $this->createForm(new DashboardType(), array('date' => $date = new \DateTime('now')));
```

Comment:
`'now'` is already the default value, no need to pass it to a `DateTime` constructor.
Why assign it to `$date` when you never use it.
And why change anything at all since `'date'` is already initialized in `DashboardType`?